tasks/eth/3-add-superchainConfig/Validation.md

```diff
@@ -60,6 +77,53 @@ There should also be a single 'State Override' in the Foundation Safe contract
   **Meaning:** Sets the `SuperchainConfigProxy` address at slot `0xfb` (251). The correctness of
    this slot is attested to in the Optimism repo at [storageLayout/L1CrossDomainMessenger.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.2.0-rc.1/packages/contracts-bedrock/snapshots/storageLayout/L1CrossDomainMessenger.json#L122-L127).
 
+### `0x5a0aae59d09fccbddb6c6cceb07b7279367c3d2a` (The 2 of 2 `ProxyAdmin` owner Safe)
```

Comment:
I do not love the `cast index` gymnastics required below, but it seemed like the most straightforward way to validate this state change without introducing tools which haven't been used in the ceremony previously.

Reply:
It was easy for me to follow so I'm good with keeping it 👍